src/EditorFeatures/CSharp/RenameTracking/CSharpRenameTrackingCodeFixProvider.cs

```diff
@@ -1,45 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Collections.Generic;
-using System.Composition;
-using Microsoft.CodeAnalysis.CodeFixes;
-using Microsoft.CodeAnalysis.Editor.Host;
-using Microsoft.CodeAnalysis.Editor.Implementation.RenameTracking;
-using Microsoft.VisualStudio.Text.Operations;
-
-namespace Microsoft.CodeAnalysis.Editor.CSharp.RenameTracking
-{
-    // TODO: Remove the ExtensionOrder attributes once a better ordering mechanism is available
-
-    [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.RenameTracking), Shared]
-    [ExtensionOrder(After = PredefinedCodeFixProviderNames.AddImport)]
-    [ExtensionOrder(Before = PredefinedCodeFixProviderNames.AddMissingReference)]
```

Comment:
note that the only 'fix' that rename comes after is `add using`.  It wants to come before everything else.  Later, i will show how we accomplish this more cleanly.

Reply:
Woah! Glad to see this hard coded list go away...